### ISSUE TITLE
Error running 'stop' command.

### ISSUE THREAD

**Issue by jaggzh** [gryphonshafer/Bot-IRC#1]
Using the example from the synopsis (modified slightly to remove the non-existing example plugins and such), I get:
$ ./foobot stop                                                                       
Odd number of elements in anonymous hash at /home/jaggz/perl5/lib/perl5/Bot/IRC.pm    ↿ line 481, <DATA> line 1.                                                            
Odd number of elements in anonymous hash at /home/jaggz/perl5/lib/perl5/Bot/IRC.pm    ↿ line 503, <DATA> line 1.                                                            
Undefined value assigned to typeglob at /home/jaggz/perl5/lib/perl5/Bot/IRC.pm line   ↿ 508, <DATA> line 1.

**Comment by gryphonshafer** [comment]
Could you post the exact code that's failing for you?

**Comment by gryphonshafer** [comment]
I suspect you're using the inline plugin structure example, which is just a structure, not actually usable as-is.

    $bot->load({
        hooks => [ [ {}, sub {}, {} ] ],
        helps => { name => 'String' },
        subs  => { name => sub {} },
        ticks => [ [ '0 * * * *', sub {} ] ],
    });

If you remove that, you'll stop seeing the warnings.

Additionally, I've released an update of Bot:IRC as version 1.21 that makes the documentation a little more clear and also notes invalid inline plugin settings.

Resolved with c63c25f.
